packages/user-operation-controller/src/UserOperationController.ts

```diff
@@ -451,6 +461,7 @@ export class UserOperationController extends BaseController<
             destinationTokenAmount: swaps.destinationTokenAmount ?? null,
             destinationTokenDecimals: swaps.destinationTokenDecimals ?? null,
             destinationTokenSymbol: swaps.destinationTokenSymbol ?? null,
+            approvalTxParams: swaps.approvalTxParams ?? null,
```

Comment:
[nit] we can move this upwards to not break the alphabetical

Reply:
Thanks for catching these!